test/metabase/models/recent_views_test.clj

```diff
@@ -4,90 +4,240 @@
    [java-time.api :as t]
    [metabase.models.recent-views :as recent-views]
    [metabase.test :as mt]
+   [metabase.util.log :as log]
    [toucan2.core :as t2]
    [toucan2.tools.with-temp :as t2.with-temp]))
 
+(def test-user :rasta)
```

Comment:
is this needed? most tests just use test user names directly inline

Reply:
Fixed